### DISCUSSION TITLE
Daily Standup 13-11-2024

### DISCUSSION THREAD

**Discussion by josealfredo79** [SistemasTecTlaxiaco/tim.shop#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by isairey** [comment]
**1.- ¿Qué hice en la última sesión?
2.- ¿Qué haré en esta sesión?
3.- ¿Qué obstáculos tengo?**

**Reply by Irvingzr16** [reply]
**1.-** Participe en la elaboración del Sprint Planning
**2.-** Se hará un breve seguimiento y comunicación del proyecto
**3.-** Hasta este momento, el único obstáculo que enfrentamos es la ineficiente conexión a internet

**Reply by isairey** [reply]
Muy bien compañero, andas muy productivo estas a nada de ser el nuevo líder de equipo

**Reply by Irvingzr16** [reply]
Espero y también aumente mi aguinaldo! 🙏
